src/Core/Layer/Layer.js

```diff
@@ -1,5 +1,77 @@
 import { EventDispatcher } from 'three';
 
+/**
+ * Layer's sequence (index) event
+ * @event Layer#sequence-property-changed
+ * @property layerId {string} the id of the layer
+ * @property new {object}
+ * @property new.sequence {number} the new value of the layer sequence
+ * @property previous {object}
+ * @property previous.sequence {number} the previous value of the layer sequence
+ * @property target {Layer} dispatched on layer
+ * @property type {string} sequence-property-changed
+ * @example
```

Comment:
I think it's quite good to have some examples in the doc. Maybe those ones are a bit too detailed, maybe I can remove the prints part and leave just how to call the function. WDYT ?